README.md

```diff
@@ -42,64 +58,75 @@ Stay updated with the latest developments on the
 
 ## Installation
 
-1. Clone or download the desired version of PECOS.
-2. Navigate to the root directory, where `pyproject.toml` is located.
-3. Install using pip:
+### Python Package
+
+To install the main Python package:
 
 ```sh
-pip install .
+pip install quantum-pecos
 ```
 
-To install optional dependencies, such as for Wasm support or state vector simulations, see `pyproject.toml` for list of
-options. To install all optional dependencies use:
+This will install both `quantum-pecos` and its dependency `pecos-rslib`.
+
+For optional dependencies:
 
 ```sh
-pip install .[all]
+pip install quantum-pecos[all]
 ```
 
-Certain simulators have special requirements and are not installed by the command above. Installation instructions for these are provided [here](#simulators-with-special-requirements).
+**NOTE:** The `quantum-pecos` package is imported like: `import pecos` and not `import quantum_pecos`.
 
-For development, use (while including installation options as necessary):
+Certain simulators have special requirements and are not installed by the command above. Installation instructions for
+these are provided [here](#simulators-with-special-requirements).
 
-On Linux/Mac:
 
-```sh
-python -m venv .venv
-source .venv/bin/activate
-pip install -U pip setuptools
-pip install -r requirements.txt
-make metadeps
-pre-commit install
-pip install -e .
-```
+### Rust Crates
 
-On Windows:
+To use PECOS in your Rust project, add the following to your `Cargo.toml`:
 
-```sh
-python -m venv .venv
-.\venv\Scripts\activate
-pip install -U pip setuptools
-pip install -r requirements.txt
-make metadeps
-pre-commit install
-pip install -e .
+```toml
+[dependencies]
+pecos = "0.x.x"  # Replace with the latest version
 ```
 
-See `Makefile` for other useful commands.
+## Development Setup
 
-Tests can be run using:
+For developers who want to contribute or modify PECOS:
 
-```sh
-pytest tests
-```
+1. Clone the repository:
+   ```sh
+   git clone https://github.com/PECOS-packages/PECOS.git
+   cd PECOS
+   ```
 
-### Simulators with special requirements
+2. Set up the development environment using the makefile (for Windows you may need to use a shell that has access to Linux commands such as utilizing [git bash](https://gitforwindows.org/)):
+   ```sh
+   make venv
+   make build
+   ```
+
+3. Run tests:
+   ```sh
+   make test
+   ```
+
+4. Run pre-commit (after [installing it](https://pre-commit.com/))
+   ```sh
+   make pre-commit
```

Comment:
I wish there was a single step to set up the virtual environment and not all make commands be calling `make venv`, which can be slow and annoying.

Reply:
The `venv` target has been removed from others.